GOVERNANCE.md

```diff
@@ -0,0 +1,86 @@
+---
+permalink: /governance/
+---
+# Governance
+
+This project uses [Martha's Rules][marthas-rules] for making decisions.  The
+first section below describes these rules; the second lists voting members of
+the project and explains how to become one, while the third lists decisions made
+to date.
+
+[marthas-rules]: https://journals.sagepub.com/doi/10.1177/088610998600100206
+
+## Making Decisions
+
+1.  Before each meeting, anyone who wishes may sponsor a proposal by filing an
+    issue in the GitHub repository tagged "proposal".  Proposals must be filed
+    at least 24 hours before a meeting in order to be considered at that
+    meeting, and must include:
+    -   a one-line summary (the subject line of the issue)
+    -   the full text of the proposal
+    -   any required background information
+    -   pros and cons
+    -   possible alternatives
+
+2.  A quorum is established in a meeting if half or more of voting members are
+    present.
+
+3.  Once a person has sponsored a proposal, they are responsible for it.  The
+    group may not discuss or vote on the issue unless the sponsor or their
+    delegate is present.  The sponsor is also responsible for presenting the
+    item to the group.
+
+4.  After the sponsor presents the proposal, a sense vote is cast for the
+    proposal prior to any discussion:
+    -   Who likes the proposal?
+    -   Who can live with the proposal?
+    -   Who is uncomfortable with the proposal?
+
+5.  If all of the group likes or can live with the proposal, it is immediately
+    accepted with no further discussion.
+
+6.  If most of the group is uncomfortable with the proposal, it is postponed for
+    further rework by the sponsor.
+
+7.  If any members are uncomfortable:
+    1.  They can briefly state their objections and tell the facilitator how
+        much time they reasonably require to make comments.
+    1.  The facilitator then sets a time limit for discussion at 10 minutes
+        plus whatever upward adjustment needed to meet any special needs
```

Comment:
the phrase 'special needs' is abelist language. I would suggest to change it to "account for any barriers to verbal communication" which refocuses the adjustment on the barriers, not the person.

Reply:
After some local consultation, I changed it to "plus whatever upward adjustment needed to account for any barriers to participation people may have", since barriers may include things like a poor internet connection.